main.go

```diff
@@ -40,6 +34,19 @@ func main() {
 			Name:   "debug",
 			EnvVar: "DEBUG",
 		},
+		cli.BoolFlag{
+			Name:   "api",
+			EnvVar: "API",
+		},
+		cli.StringFlag{
+			Name:   "api-address",
+			EnvVar: "API_ADDRESS",
+			Value:  "0.0.0.0:8080",
+		},
+		cli.BoolTFlag{
```

Comment:
Shouldn't these be something like `enable-controller` to be more descriptive, as it's a flag not a "command"?